### ISSUE TITLE
Problem With PTFXs

### ISSUE THREAD

**Issue by meltynetv2** [Scullyy/scully_emotemenu#1]
Hello, so I am using the latest version of the emote menu, but when trying to use any emote with particle effects and pressing the button to start said particle effects gives the following error: https://i.gyazo.com/2ce04123addb79fb26fd0dce11a35713.png

As well I printed the asset string to the console and it does come up nil.

If any help can be given or we can find a solution, much would be appreciated!

**Comment by meltynetv2** [comment]
> I have the same message, i guess they will fix in the next update

Okay, I am glad I am not the only one. Didn't see anything, so I thought I would make this issue report.

**Comment by Cyril-Beeckman** [comment]
Can confirm

**Comment by jaystar84** [comment]
is fixed?

**Comment by meltynetv2** [comment]
Hello! I do not believe this is fixed. I tried again after the 1.0.3 update, but still get this error when pressing the keybind('g') for the effects:
https://i.gyazo.com/232629c2b08d6e1021808e6c91a09900.png

I am initiating the pee animation both through /e and through the menu and neither seem to work.

I have tried seeing why it's getting nil instead of the value suppose to be passed. I have a feeling the localplayer statebags are not being set correctly, but from what I see you are doing nothing wrong with the setting... So I am personally clueless unless I had more time to look into it myself.
